packages/core/upload/admin/src/components/MediaLibraryInput/index.js

```diff
@@ -100,8 +102,24 @@ export const MediaLibraryInput = ({
   };
 
   const handleAssetDrop = (assets) => {
-    setDroppedAssets(assets);
-    setStep(STEPS.AssetUpload);
+    const allowedAssets = getAllowedFiles(fieldAllowedTypes, assets);
+
+    if (allowedAssets.length > 0) {
+      setDroppedAssets(allowedAssets);
+      setStep(STEPS.AssetUpload);
+    } else {
+      toggleNotification({
+        type: 'warning',
+        timeout: 4000,
+        message: {
+          id: 'input.notification.not-supported',
+          defaultMessage: `You can't upload this type of file, only the following types are accepted – ${fieldAllowedTypes.join(
```

Comment:
I think we are always defaulting to the default message here because we should use `getTrad` from `utils` 👀
but I guess we'll have an issue with the `toggleNotification`, it doesn't seem to support variables in the trad 🤔

Reply:
_ahhhhh_ that's why. thank you